libs/csm/rest/csm_rest_iamuser.py

```diff
@@ -783,7 +784,7 @@ def get_random_caps(self):
         return random_cap[:-1]
 
     @RestTestLib.authenticate_and_login
-    def list_iam_users_rgw(self, max_entries=None, marker=None):
+    def list_iam_users_rgw(self, max_entries=None, marker=None, auth_header=False):
```

Comment:
it should auth_header = None
If auth_header is not None:
     header = auth_header
else
    header  = self.header

Reply:
modified accordingly